build/NuSpecs/AppxManifest.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this first line showing up as modified?  Did the Byte Order Mark (BOM) get changed unexpectedly?

Reply:
Fixed this!